guides.md

```diff
@@ -237,6 +239,140 @@ members. Skip the steps below at your discretion for low-stakes topics.
   1. Use the private dial-in number from our [technical Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
   2. _After_ prompt, enter conference code `307 314 3734 #`
 
+## Onboarding Checklist
+
+This is a checklist for onboarding a new Member to the Co-operative 🚀
+
+### Setting up communications
+
+1. Get Signal number and add to group.
+
+1. Get Matrix account and invite to spaces:
+  - Private chat `#hyphacoop-private:tomesh.net`
+  - Public chat `#hyphacoop-open:tomesh.net`
+  - Community `+hyphacoop:tomesh.net`
+
+1. Set up [`hypha.coop` email](https://link.hypha.coop/email):
+  - Create new mailbox with Mailcow
+  - Set up initial forwarder
+  - Add to `members@`
+
+### Setting up virtual office
+
+1. Invite to [Passbolt](https://pass.hypha.coop):
+  - After acceptance, add to `Member-Worker` group
+
+1. Invite to [GitHub Organization](https://github.com/hyphacoop):
+  - Add to `Member-Worker` [GitHub Team](https://link.hypha.coop/teams)
+
+1. Get Google-friendly email address:
+  - Share access to [Drive](https://link.hypha.coop/drive)
+  - Invite to calendars:
+    - https://link.hypha.coop/calendar
+    - https://link.hypha.coop/availability
+
+1. Invite to [Loomio](https://loomio.hypha.coop/).
+
+1. Share [weekly schedule](https://link.hypha.coop/schedules) (recommend without password).
+
+### Setting up employee record and payroll
+
+1. Collect information for [employee record](https://link.hypha.coop/employees).
+
+1. Add Member SIN to Passbolt and share access with `People Operations` group.
```

Comment:
Just FYI: I was giving the group admin access, and the individual read access to give introspection into the fact that their SIN was being stored in the password vault (decided not to give them write perm, as they shouldn't be able to delete their SIN from the company system)

Reply:
```suggestion
1. Add Member SIN to Passbolt and share access with `People Operations` group.
    Change the Member access to the entry as read-only, as this information is for recording keeping and the Member should not be able to modify it.
```